src/view/reviewManager.ts

```diff
@@ -189,13 +189,18 @@ export class ReviewManager {
 		GitHubCreatePullRequestLinkProvider.registerProvider(this._disposables, this, this._folderRepoManager);
 	}
 
-	private updateBaseBranchMetadata(oldHead: Branch, newHead: Branch) {
+	private async updateBaseBranchMetadata(oldHead: Branch, newHead: Branch) {
 		if (!oldHead.commit || (oldHead.commit !== newHead.commit) || !newHead.name || !oldHead.name || (oldHead.name === newHead.name)) {
 			return;
 		}
 
-		const githubRepository = this._folderRepoManager.gitHubRepositories.find(repo => repo.remote.remoteName === oldHead.upstream?.remote);
+		let githubRepository = this._folderRepoManager.gitHubRepositories.find(repo => repo.remote.remoteName === oldHead.upstream?.remote);
 		if (githubRepository) {
+			let metadata = await githubRepository.getMetadata();
```

Comment:
Hey @amunger, this should be const


```suggestion
			const metadata = await githubRepository.getMetadata();
```